web_generator/test/integration/interop_gen/ts_typing_input.d.ts

```diff
@@ -0,0 +1,17 @@
+export declare const myString: string;
```

Comment:
Maybe have an example where we do a `typeof` of a declaration that is defined afterwards?